client/src/components/Button.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+
+interface Props {
+  variant: string;
+  disabled: boolean;
+  children: any;
+}
+
+const Button: React.FC<Props> = ({ children, variant, disabled }) => {
+  if (variant === "filled" && disabled == false) {
```

Comment:
Use triple equal sign to check equality (`disabled === false`) or better yet use boolean logic (`!disabled`)